web3j/src/main/resources/config/application.yml

```diff
@@ -1,13 +1,14 @@
 workload:
   model: OPEN  # either OPEN or CLOSED workload model
   transactions: 1_000 # total transactions to be done
-  load-factor: 20 # arrival rate for OPEN model & concurrency for CLOSED model
+  load-factor: 10 # arrival rate for OPEN model & concurrency for CLOSED model
   progress-interval: 5s # interval for updating progress in web UI
 
 web3j:
+  use-grpc: false # set true for grpc
   ethClient:
-    url: # ethereum client endpoint (Note: Leave it blank to connect to embedded EVM.)
```

Comment:
It won't look clean to enter http url and do gRPC . So I suggest remove URL and have following fields.
- protocol (default: grpc) - comment: either http or grpc
- host (default: localhost) - comment: ethereum client node
- port (default: 50505) - comment: 8545 for http, 50505 for gRPC
- tls (default: false) - comment: enable or disable tls

Reply:
yeah it looks cleaner, will do